pillar.example

```diff
@@ -45,24 +45,27 @@ postgresql:
     shared_buffers: 128MB
     maintenance_work_mem: 64MB
     effective_cache_size: 4GB
+  
+  # These lists will be written in the pg_hba.conf
+  # Accepted shape:   
+  #  - ['host/local', 'DATABASE',  'USER',  'ADDRESS', 'METHOD']
+
+  acls:
+    - ['host', 'triebwerk', 'deploy', '10.14.38.0/24', 'trust']
+    - ['host', 'triebwerk', 'user_wwith_password', '10.14.38.0/24', 'password']
+    - ['host', 'repliction', 'replicant', '10.14.38.0/24', 'trust']
```

Comment:
Mmh, why was this changed? There's no feature added and it's less readable this way and more error prone (e.g. when there's a line that doesn't have the address column)

Reply:
Now you can have multiple rules for one user, as requested yesterday